### PR TITLE
Format parameter

### PR DESCRIPTION
**Pull request by GeekTrainer**
Added a parameter to handle [markdown issue](https://github.com/Microsoft/BotFramework-WebChat/issues/374) for sent messages. Set the default to be plain rather than markdown.

### CODE REVIEW

File: src/Store.ts [hunk 1]
```diff
@@ -56,6 +56,7 @@ export const shell: Reducer<ShellState> = (
 
 export interface FormatState {
     locale: string,
+    messageTextFormat: string,
```
Comment (billba) [inline src/Store.ts]: this should go in `ConnectionState`, with `user`.

File: src/Store.ts [hunk 2]
```diff
@@ -76,11 +77,15 @@ export type FormatAction = {
 } | {
     type: 'Set_Measurements',
     carouselMargin: number
+} | {
+    type: 'Set_Text_Format_Option',
+    messageTextFormat: string
```
Comment (billba) [inline src/Store.ts]: if it goes in `ConnectionState` I think you can piggyback on an existing action.

**Comment by GeekTrainer** [comment]
Changed the default to be plain.

**Comment by GeekTrainer** [comment]
And again

**Pull request merged**